tests/utils.py

```diff
@@ -0,0 +1,29 @@
+from unittest import mock
+
+from src.masonite.orm.connections.ConnectionFactory import ConnectionFactory
+from src.masonite.orm.connections.MySQLConnection import MySQLConnection
+from src.masonite.orm.connections.SQLiteConnection import SQLiteConnection
+
+
+class MockMySQLConnection(MySQLConnection):
```

Comment:
How does this mocking stuff work?

Reply:
I do this to not fire the query.
It is a normal mocking of the `make_connection` function. 
This method will return a mock.Magick that help to not fire the query on the database really.